base/sparse/sparsematrix.jl

```diff
@@ -292,7 +292,10 @@ function copy!(A::SparseMatrixCSC, B::SparseMatrixCSC)
     return A
 end
 
-similar(S::SparseMatrixCSC, Tv::Type = eltype(S)) = SparseMatrixCSC(S.m, S.n, copy(S.colptr), copy(S.rowval), Vector{Tv}(length(S.nzval)))
+function similar(S::SparseMatrixCSC, ::Type{Tv} = eltype(S)) where {Tv}
```

Comment:
Unfortunately I don't know how to fix this without opening an other pull request :( I just edited the file in Github... I hope that's not a huge issue.